src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -478,7 +576,6 @@ ov::npuw::LLMCompiledModel::LLMCompiledModel(const std::shared_ptr<ov::Model>& m
     LOG_DEBUG("6.Check and apply opt layout if applicable.");
 
     const bool optimize_v_tensors = m_cfg.get<::intel_npu::NPUW_LLM_OPTIMIZE_V_TENSORS>();
-    // NB: Try to apply opt transpose only for Llama-2-7b-chat-hf model
     if (optimize_v_tensors) {
         if (optimize_value_tensors(kvcache_model)) {
```

Comment:
Can we add debug log to check if pass was applied?

Reply:
@TolyaTalamanov please check if that is OK